<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIAntiSpam</class>
 <widget class="QDialog" name="UIAntiSpam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AntiSpam configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="11">
    <widget class="QGroupBox" name="groupBox_PHRASE">
     <property name="title">
      <string>AntiSpam</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1" colspan="4">
       <widget class="LineEdit" name="lineEdit_PHRASE">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="LineEdit" name="lineEdit_KEY">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Phrase</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Key</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="spinBox_TRYCOUNT">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Attempts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0" colspan="7">
    <widget class="QCheckBox" name="checkBox_ASENABLE">
     <property name="text">
      <string>Enable AntiSpam</string>
     </property>
    </widget>
   </item>
   <item row="7" column="10">
    <widget class="QPushButton" name="pushButton_OK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkBox_ASFILTER">
     <property name="text">
      <string>Work as filter</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_BLACK">
     <property name="title">
      <string>Black list</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="LineEdit" name="lineEdit_BLACK"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_ADDBLACK">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_REMBLACK">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_CLRBLACK">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QTreeWidget" name="treeWidget_BLACK">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>Users</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3" colspan="4">
    <widget class="QGroupBox" name="groupBox_GRAY">
     <property name="title">
      <string>Gray List</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="LineEdit" name="lineEdit_GRAY"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_ADDGRAY">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_REMGRAY">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_CLRGRAY">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QTreeWidget" name="treeWidget_GRAY">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>Users</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="7" colspan="4">
    <widget class="QGroupBox" name="groupBox_WHITE">
     <property name="title">
      <string>White list</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="LineEdit" name="lineEdit_WHITE"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_ADDWHITE">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_REMWHITE">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButton_CLRWHITE">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QTreeWidget" name="treeWidget_WHITE">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>Users</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="pushButton_GTOW">
     <property name="toolTip">
      <string>Move user to &quot;White&quot; list</string>
     </property>
     <property name="text">
      <string>&gt;&gt;&gt;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_BTOG">
     <property name="toolTip">
      <string>Move user to &quot;Gray&quot; list</string>
     </property>
     <property name="text">
      <string>&gt;&gt;&gt;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QPushButton" name="pushButton_WTOG">
     <property name="toolTip">
      <string>Move user to &quot;Gray&quot; list</string>
     </property>
     <property name="text">
      <string>&lt;&lt;&lt;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pushButton_GTOB">
     <property name="toolTip">
      <string>Move user to &quot;Black&quot; list</string>
     </property>
     <property name="text">
      <string>&lt;&lt;&lt;</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_BTOW">
     <property name="toolTip">
      <string>Move user to &quot;White&quot; list</string>
     </property>
     <property name="text">
      <string>&lt;&lt;&lt;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="10">
    <widget class="QPushButton" name="pushButton_WTOB">
     <property name="toolTip">
      <string>Move user to &quot;Black&quot; list</string>
     </property>
     <property name="text">
      <string>&gt;&gt;&gt;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="checkBox_FILTER_OPS">
     <property name="text">
      <string>Filter Operators</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
